src/material/button/fab.scss

```diff
@@ -11,7 +11,7 @@
 @use '../core/tokens/m2/mdc/extended-fab' as m2-mdc-extended-fab;
 @use '../core/tokens/m2/mdc/fab' as m2-mdc-fab;
 
-@include mdc-custom-properties.configure($emit-fallback-values: false, $emit-fallback-vars: false) {
+@include mdc-custom-properties.configure($emit-declaration-fallback: false, $emit-fallback-vars: false) {
```

Comment:
Can you show an example of what fall back this is referring to? Is it the fallback of the variable itself (e.g. `var(--foo, #fallback)` or MDC's IE11 fallbacks like: `color: red; color: var(--foo, red);`?

Reply:
The issue will be that MDC's new theming system doesn't have this configuration option to skip variable fallbacks at all. This PR helps line us up for the next switch to their new mixins.

Reply:
I believe I understand your fix, which is to set a default variable value anyways. But that will be extra tech debt we'll want to remove anyways